tsc/data/translations/fr.po

```diff
@@ -1,1605 +1,1728 @@
-# This file is distributed under the same license as the Secret Maryo Chronicles package.
-#
-# Copyright © 2008 Tester <joelekiffeur@yahoo.fr>
-# Copyright © 2008 Dylan Aïssi <bob.dybian@gmail.com>
-#
-# Translation of Secret Maryo Chronicles to French
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR The TSC Contributors
```

Comment:
Do not remove copyright statements of earlier workers! Please re-add them, and then add giby’s copyright. It has to look likt his finally:

~~~~~~~~~~~~~~~~~~~~~~~~
Copyright © 2008 Tester <joelekiffeur@yahoo.fr>
Copyright © 2008 Dylan Aïssi <bob.dybian@gmail.com>
Copyright © 2015 Benjamin Leduc <(email if he wants that)>
~~~~~~~~~~~~~~~~~~~~~~~~

And of course, remove the placeholders.

Vale,
Quintus
~~~~~~~~~~~~~~~~~~~~~~~~